agent/agent.go

```diff
@@ -188,14 +178,25 @@ func (a *Agent) handleDeploy(m *nats.Msg) {
 		return
 	}
 
-	_ = a.workAck(m, true, "Workload accepted")
+	_ = a.workAck(m, true, "Workload deployed")
```

Comment:
How are we saying "Workload deployed" here if the next line is for deploying the workload?

Reply:
Good catch, will move this below `a.provider.Deploy()`.